tests/baselines/reference/ModuleWithExportedAndNonExportedFunctions.errors.txt

```diff
@@ -1,4 +1,4 @@
-tests/cases/conformance/internalModules/exportDeclarations/ModuleWithExportedAndNonExportedFunctions.ts(28,13): error TS2339: Property 'fn2' does not exist on type 'typeof A'.
+tests/cases/conformance/internalModules/exportDeclarations/ModuleWithExportedAndNonExportedFunctions.ts(28,13): error TS2551: Property 'fn2' does not exist on type 'typeof A'. Did you mean 'fng'?

```

Comment:
After discussing what we want from this feature with @RyanCavanaugh, here's how we decided to summarise it: suggest corrections for misspellings that you "can't see".

That means we want very high precision (no false positives) and don't care about recall (it's ok to miss some fixes). It also means that short identifiers should only suggest a few things. Specifically, 3-character identifiers should only suggest additions, basically to cover the `foo` &rarr; `_foo` / `foo1` case. `fn2` &rarr; `fng` and `bar` &rarr; `baz` are obvious.